src/routers/nomination/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { AuthenticatedRequest } from '../../types/network';
+
+import express, { Response, NextFunction } from 'express';
+import auth from '../../middleware/auth';
+import validate from '../../middleware/validate';
+import NominationModel from '../../models/nomination';
+import getValidations from './validations';
+import { LocalRoutes } from './types';
+
+const router = express.Router();
+
+router.post(
+  '/',
+  getValidations(LocalRoutes.CREATE_NOMINATION),
```

Comment:
I like this but I wish we could use `LocalRoutes` for the route path as well :( like we have for top-level routes.

If we swapped out the description for the route path and added the description in a comment we would still have the issue of POST, GET and all the others sharing the same path :'(

Reply:
Actually that's why we want to use descriptions _instead_ of route paths. If we use descriptions, then regardless of whether the route was `POST`, `PATCH` etc, we would guarantee uniqueness.

What do you think?

Reply:
Yep I'm def for having each route maintain its validation. And using the description makes sense in that way too!

I just kinda wished we could have the route path information in the enum too somehow. but I dream 😢